components/eamxx/tests/multi-process/dynamics_physics/mam/homme_shoc_cld_spa_p3_rrtmgp_mam4_wetscav/CMakeLists.txt

```
@@ -70,6 +70,7 @@ set (TEST_INPUT_FILES
   scream/init/spa_init_ne2np4.nc
   scream/init/spa_file_unified_and_complete_ne2np4L72_20231222.nc
   scream/init/${EAMxx_tests_IC_FILE_72lev}
+  scream/init/scream_unit_tests_aerosol_optics_ne2np4L72_20220822.nc
```

Can we use `scream/init/${EAMxx_tests_IC_FILE_MAM4xx_72lev}`
```suggestion
  scream/init/${EAMxx_tests_IC_FILE_MAM4xx_72lev}
```
It is a different file, so we might need to bless it, but it will be consistent with the rest of the testing.

---

That's fine with me. I'll push the change.